buildlib/azure-pipelines.yml

```diff
@@ -23,6 +23,9 @@ stages:
       - job: commit_title
         displayName: commit title
         steps:
+          - checkout: self
```

Comment:
do we have to specify it for every job? no any "global" setting?

Reply:
Yes, in order to have it explicitly set, we need it for every job since each job starts with checkout (it was implicit). But we might not need it for every job we have, maybe just for MLNX pool jobs.